doc/source/index.rst

```diff
@@ -1,14 +1,42 @@
-==========================================
-Ansys Sphinx Theme documentation |version|
-==========================================
+Ansys Sphinx Theme documentation
+################################
 
-.. include:: ../../README.rst
+Version: |version|
```

Comment:
Why not have the version in the title like our other documentation? Having it here seems odd--and makes it stand out less.